android/lib/src/androidTest/java/com/ern/api/impl/BasicNavigationApiTest.java

```diff
@@ -0,0 +1,409 @@
+package com.ern.api.impl;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.lifecycle.Lifecycle;
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.ext.junit.rules.ActivityScenarioRule;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.filters.LargeTest;
+
+import com.ern.api.impl.navigation.ElectrodeBaseActivity;
+import com.ern.api.impl.navigation.MiniAppNavigationFragment;
+import com.ernnavigationApi.ern.api.EnNavigationApi;
+import com.ernnavigationApi.ern.model.ErnNavRoute;
+import com.ernnavigationApi.ern.model.NavigationBar;
+import com.walmartlabs.electrode.reactnative.bridge.ElectrodeBridgeResponseListener;
+import com.walmartlabs.electrode.reactnative.bridge.FailureMessage;
+import com.walmartlabs.electrode.reactnative.bridge.None;
+import com.walmartlabs.electrode.reactnative.bridge.helpers.Logger;
+import com.walmartlabs.ern.container.ElectrodeReactContainer;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Objects;
+import java.util.concurrent.CountDownLatch;
+
+import static com.ern.api.impl.BasicNavigationApiTest.SampleActivity.ROOT_COMPONENT_NAME;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.fail;
+
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+public class BasicNavigationApiTest {
+
+    private static final String TITLE_ROOT_PAGE = ApplicationProvider.getApplicationContext().getString(com.walmartlabs.ern.navigation.test.R.string.root_page_title);
+    private static final String TITLE_PAGE_1 = "page 1";
+    private static final String COMPONENT_PAGE_1 = "Component1";
+
+    @Before
+    public void setUp() {
+        Logger.overrideLogLevel(Logger.LogLevel.VERBOSE);
+    }
+
+    @Rule
+    public ActivityScenarioRule<SampleActivity> rule = new ActivityScenarioRule<SampleActivity>(SampleActivity.class);
+
+    @Test
+    public void testRootComponentRendering() {
+        Assert.assertNotNull(rule);
```

Comment:
Best practice to use static imports for things like `assertNotNull` etc. (Just like you've already added a static import for `fail`) - It makes these lines shorter and easier to read.